### ISSUE TITLE
100 submissions/day limit

### ISSUE THREAD

**Issue by RuRo** [soachishti/moss.py#58]
```
UPDATES

Nov 13, 2022, Due to persistent use of robots to submit tens of thousands of Moss jobs per day, the submission limit of 100 submissions/day/user is now being enforced.
```

Does anybody know if "100 submissions" refer to 100 files or 100 "runs" of the script?

I know that my script is correct, because I already successfully processed a couple of our tasks, but now I keep getting all kind of errors:
- `[Errno -3] Temporary failure in name resolution`
- `[Errno -5] No address associated with hostname`
- `[Errno 101] Network is unreachable`
- `[Errno 104] Connection reset by peer`
- `[Errno 111] Connection refused`
- sometimes the script just hangs for hours, and I am forced to kill it

So now I am wondering if I am being throttled/banned. I am 99.9% sure, that I didn't make more than 100 individual submissions, because my script sleeps for 30 minutes between each retry, and so it shouldn't have exceeded 48 submissions per day (assuming that "submission" means a single batch of files and not each individual file).

I seriously don't understand, why is the Moss server so flaky. The official explanation is that there is a lot of load on the servers, which is understandable, but if 90% of submission attempts end in failures due to the flaky server, then most of that load comes from retrying the same submission over and over again instead of actually processing new submissions.

**Comment by mplumleeUNH** [comment]
I'm pretty sure it's 100 runs of the script--we ran about 10 runs with 90 student submissions each a few days ago without issue. The site appears to be down today (can't get to our results). This has happened in the past and the site has usually come back within a couple of days. It is such a lifesaver, I wish we could pay for a more stable or locally-hosted version!

**Comment by RuRo** [comment]
Yeah, it seems that the `http://moss.stanford.edu/` server is offline like 90% of the time, which is extremely weird, because it's only the HTTP host that is down. Most of the time the `moss.stanford.edu:7690` submission server is up and working fine and returns urls to the HTTP server, but you can't view or download the results, because the HTTP server is down. I would have expected that simply serving the already generated `html` files would put less load on the server (compared to the submission process).
